ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -3676,6 +3676,48 @@ def test_torch_instance_cumsum(
     )
 
 
+# cumsum_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="cumsum_",
+    dtype_value=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        shape=st.shared(helpers.get_shape(min_num_dims=1), key="shape"),
+    ),
+    dim=helpers.get_axis(
+        shape=st.shared(helpers.get_shape(), key="shape"),
+        allow_neg=True,
+        force_int=True,
+    ),
+)
+def test_torch_instance_cumsum_(
+    dtype_value,
+    dim,
+    dtypes,
```

Comment:
seems like this is already generated in `dtype_value`

Reply:
@guillesanbri So we need to remove dtype_value from the argument right?

Reply:
Actually, `dtype` has to be removed from the args. You are generating `dtype_value` and `dim` in the decorator (which is fine), but the function is expecting `dtype`, `dim` and `dtypes`